### DISCUSSION TITLE
How to open a URL with the keyboard?

### DISCUSSION THREAD

**Discussion by nzig** [wezterm/wezterm#1286] [Q&A] [answered]
I'd like to be able to open a URL without using the mouse. I can copy the URL with Quick Select Mode, but I could not find a way to trigger opening the URL with just the keyboard.

Ideally I could use some modifier like Ctrl+prefix in Quick Select Mode to open the URL. Is there a way to configure this?

**Comment by wez** [comment, marked as answer]
The latest nightly build allows this using quickselect to trigger a lua callback which in turn can then open the URL.
There's an example in the docs:
https://wezfurlong.org/wezterm/config/lua/keyassignment/QuickSelectArgs.html

**Reply by gegoune** [reply]
This is amazing, great thanks! One question, when I run that function I get:

```
Select:   (type highlighted prefix to copy, uppercase pastes, ESC to cancel)
```
which is not applicable for that use case. Can that string be changed?

**Reply by wez** [reply]
what sort of change would you like to see there?

**Reply by gegoune** [reply]
Something truthful, I know what I invoked so it's not a big problem really, but that default message is just not true in this case. Maybe something like
```
Select:   (type highlighted prefix to open url, ESC to cancel)
```

Don't want to waste your time on something as irrelevant as this.

**Reply by wez** [reply]
https://github.com/wez/wezterm/commit/38087755e1ccef9637a4b2a83e349fce9386268f

**Reply by gegoune** [reply]
Amazing, thanks! Will check it out once new nightly is available.
